docs/draft_production_docs/oadp-self-service.md

```diff
@@ -0,0 +1,484 @@
+
+1. The NonAdminBackup creates the Velero Backup CR instance in a secure way that limits the users access.
+2. A user cannot specify the namespace that will be backed up.  The namespace from which the NAB object is created is the defined namespace to be backed up.
+3. In addition to the creation of the Velero Backup the NonAdminBackup object's main purpose is to track the status of the Velero Backup in a secure and clear way.
+
+### NonAdminBackup NAB Example:
+
+```
+apiVersion: oadp.openshift.io/v1alpha1
+kind: NonAdminBackup
+metadata:
+  name: mybackup-1
+  namespace: nacuser1
+spec:
+  backupSpec:
+    snapshotMoveData: true
+```
+
+Once created the NAB will look similar to the following:
+
+```
+apiVersion: oadp.openshift.io/v1alpha1
+kind: NonAdminBackup
+metadata:
+  creationTimestamp: "2025-02-21T20:57:35Z"
+  finalizers:
+  - nonadminbackup.oadp.openshift.io/finalizer
+  generation: 2
+  name: mybackup-1
+  namespace: nacuser1  <--- The namespace is set by the NAC controller
+  resourceVersion: "20714121"
+  uid: 93effb39-9762-4d04-8e9e-194ebe6b9b31
+spec:
+  backupSpec:
+    snapshotMoveData: true  
+status:
+  conditions:
+  - lastTransitionTime: "2025-02-21T20:57:35Z"
+    message: backup accepted  <--- The NAC controller reconciled the NAB object and created the Velero Backup object
+    reason: BackupAccepted
+    status: "True"
+    type: Accepted
+  - lastTransitionTime: "2025-02-21T20:57:35Z"
+    message: Created Velero Backup object
+    reason: BackupScheduled
+    status: "True"
+    type: Queued
+  phase: Created  <--- The NAB object is in the Created phase
+  queueInfo:
+    estimatedQueuePosition: 0  <--- The NAB object status in the queue, once complete it is set to 0
+  veleroBackup:
+    nacuuid: nacuser1-mybackup-1-588ce989-387e-4352-b04a-1fd6b7712370  <--- The NAC controller created the Velero Backup object and set the nacuuid
+    name: nacuser1-mybackup-1-588ce989-387e-4352-b04a-1fd6b7712370 <--- The associated Velero Backup name
+    namespace: openshift-adp
+    status:  <--- The status of the Velero backup object displayed by the NAB object
+      backupItemOperationsAttempted: 3
+      backupItemOperationsCompleted: 3
+      completionTimestamp: "2025-02-21T20:59:28Z"
+      expiration: "2025-03-23T20:57:35Z"
+      formatVersion: 1.1.0
+      hookStatus: {}
+      phase: Completed  <--- The Velero backup object is in the Completed phase, successful
+      progress:
+        itemsBackedUp: 57
+        totalItems: 57
+      startTimestamp: "2025-02-21T20:57:35Z"
+      version: 1
+```
+The complete nonAdminBackup resource definition can be found here: [NonAdminBackup CRD](https://github.com/openshift/oadp-operator/blob/master/bundle/manifests/oadp.openshift.io_nonadminbackups.yaml)
+
+### NonAdminRestore NAR Example:
+
+```
+apiVersion: oadp.openshift.io/v1alpha1
+kind: NonAdminRestore
+metadata:
+  name: example
+  namespace: nacuser1
+spec: 
+  restoreSpec:
+    backupName: mybackup-1
```

Comment:
It's important to mention this is the reference to local Non Admin Backup and not the Velero backup object.

Reply:
+1